src/metabase/api/table.clj

```diff
@@ -48,6 +48,12 @@
     (t2/hydrate tables :db)
     (filterv mi/can-read? tables)))
 
+(api/defendpoint GET "/uploaded"
+  "Get all `Tables` visible to the current user which were created by uploading a file."
+  []
+  (as-> (t2/select Table, :active true, :is_upload true, {:order-by [[:name :asc]]}) tables
```

Comment:
I'm not fond of coupling to the "internal" model detail that it uses an `is_upload` field, but this full query doesn't belong in `metabase.upload`. In general we don't seem that concerned with model encapsulation, so guessing this is fine.

Reply:
yeah, I share your hesitation but this is very much How Things Are Done here so we may as well keep on keeping on.